README.md

```diff
@@ -11,7 +11,9 @@ Treat others as you wish to be treated.
 We are a team, we don't judge rather support each other.
 
 ## Autonomy
-Each member of the team is the owner of their tasks. They choose how to do their tasks, when to ask for help, and when to pull the plug on a discussion that they asked for.
+Each member of the team is the owner of their tasks. Together with the accountable person, they ensure that the solution of the tasks is aligned to the goals. They are responsible for ensuring that the task meets the quality and the standards that are needed, determining when and how best to involve others. 
```

Comment:
```suggestion
Each member of the team is the owner of their tasks. Together, with the accountable person, ensure that the solution of the task is aligned to the goals. They are responsible for ensuring that the solution meets the quality and the standards needed, determining when and how best to involve others.
```

I made some minor adjustments here. Feel free to change them or keep them, however you think sound best.